config.yml

```diff
@@ -6,66 +6,76 @@ CI_HELPER_VERSION: &CI_HELPER_VERSION 0.0.6
 ci_helper_test: &ci_helper_test ci-helper version -e
 MODD_VERSION: &MODD_VERSION 0.5
 
+PYTHON_MANAGED_VERSION: &PYTHON_MANAGED_VERSION
+  - 3.6
+  - 3.7
+  - 3.8
+
+ansible_tests: &ansible_tests
+    cmd:
+      - python --version
+      - pip --version
+      - ansible --version
+      - ansible-playbook --version
+
 ansible:
-  "2.5.5":
+  "2.8":
+    multiple_based_version: true
+    template_vars:
+      VERSION: *PYTHON_MANAGED_VERSION
     build_args:
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
-      VERSION: 2.5.5
-    test_config:
-      cmd:
-        - *ci_helper_test
-        - ansible --version
-        - ansible-playbook --version
-        - python --version
+      ANSIBLE_VERSION: 2.8.8
+    test_config: *ansible_tests
+  "2.9":
```

Comment:
Is there any reason to maintains 2 version of ansible image ? 
The last one's not enough ? (previously we have only one version : 2.5.5)

Reply:
None, that's true. But we need at least one tag of 2.8. I will keep 2.8 for the PR and switch to 2.9 after